sources/assets/bloodhound/customqueries.json

```diff
@@ -46,87 +46,108 @@
             "queryList": [{
                 "final": true,
                 "query": "MATCH p=shortestPath((c {owned: true})-[*1..3]->(s)) WHERE NOT c = s RETURN p"
             }]
         },
+        {
+            "name": "Owned users with permissions against GPOs",
+            "category": "Owned Objects",
+            "queryList": [{
+                "final": true,
+                "query": "MATCH p=(u:User {owned:true})-[r:AllExtendedRights|GenericAll|GenericWrite|Owns|WriteDacl|WriteOwner|GpLink*1..]->(g:GPO) RETURN p"
+            }]
+        },
         {
             "name": "Connections between different domains/forests",
-            "category": "Tigers love pepper",
+            "category": "Domains/Forests",
             "queryList": [{
                 "final": true,
                 "query": "MATCH p = (a)-[r]->(b) WHERE NOT a.domain = b.domain RETURN p"
             }]
         },
         {
             "name": "Connections (ACEs only) between different domains/forests",
-            "category": "Tigers love pepper",
+            "category": "Domains/Forests",
             "queryList": [{
                 "final": true,
                 "query": "MATCH p = (a)-[r]->(b) WHERE NOT a.domain = b.domain AND r.isacl = True RETURN p"
             }]
         },
         {
-            "name": "Owned users with permissions against GPOs",
-            "category": "Tigers love pepper",
+            "name": "Can a user from domain ‘A ‘ do anything to any computer in domain ‘B’ (Warning: VERY Heavy)",
+            "category": "Domains/Forests",
             "queryList": [{
-                "final": true,
-                "query": "MATCH p=(u:User {owned:true})-[r:AllExtendedRights|GenericAll|GenericWrite|Owns|WriteDacl|WriteOwner|GpLink*1..]->(g:GPO) RETURN p"
-            }]
+                    "final": false,
+                    "title": "Select source domain...",
+                    "query": "MATCH (n:Domain) RETURN n.name ORDER BY n.name DESC"
+                },
+                {
+                    "final": false,
+                    "title": "Select destination domain...",
+                    "query": "MATCH (n:Domain) RETURN n.name ORDER BY n.name DESC"
+                },
+                {
+                    "final": true,
+                    "query": "MATCH (n:User {domain: {result}}) MATCH (m:Computer {domain: {}}) MATCH p=allShortestPaths((n)-[r:MemberOf|HasSession|AdminTo|AllExtendedRights|AddMember|ForceChangePassword|GenericAll|GenericWrite|Owns|WriteDacl|WriteOwner|CanRDP|ExecuteDCOM|AllowedToDelegate|ReadLAPSPassword|Contains|GpLink|AddAllowedToAct|AllowedToAct|SQLAdmin*1..]->(m)) RETURN p",
+                    "startNode": "{}",
+                    "allowCollapse": false
+                }
+            ]
         },
         {
             "name": "Kerberoastable users with a path to DA",
-            "category": "They hate cinnamon",
+            "category": "Kerberoasting",
```

Comment:
Can you switch "Kerberoasting" to "Roasting" as there are other techniques involved (e.g. ASREProasting)?